agent/php_execute.c

```diff
@@ -1982,7 +1991,7 @@ static void nr_php_instrument_func_begin(NR_EXECUTE_PROTO) {
           "Uncaught exception ", &NRPRG(exception_filters) TSRMLS_CC);
       php_observer_clear_uncaught_exception_globals();
     }
-  } else {
+  } else if (NULL != NRPRG(txn)) {
```

Comment:
This seems to be unnecessary - it's already checked on line [1938](https://github.com/zsistla/newrelic-php-agent/blob/txn_check/agent/php_execute.c#L1938).
```suggestion
  } else {
```